src/Pages/NotFoundPage/NotFoundPageStyled.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import FONTS from '../../Styles/fontsStyledComponents';
+const NotFoundPageWrapper = styled.div`
+  margin: 51px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .mainContentWrapper {
+    width: 16.25rem;
+    height: 7.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 35px 0 68px 0;
```

Comment:
I think there is no need for last value. Maybe You should keep it 35px 0 68px ?

Reply:
For me my version it's more readable